common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -313,6 +316,54 @@ def test_from_xml(self):
             'transcripts': {'uk': 'ukrainian_translation.srt', 'de': 'german_translation.srt'},
         })
 
+    @ddt.data(
+        ('course-v1:test_course+test_num+test_run',
+         '/asset-v1:test_course+test_num+test_run+type@asset+block@test.png'),
+        ('test_course/test_num/test_run', '/c4x/test_course/test_num/asset/test.png')
+    )
+    @ddt.unpack
+    def test_from_xml_when_handout_is_course_assest(self, course_id_string, expected_handout_link):
+        """
+        Test that if handout link is course_assest then it will contain targeted course_id in handout link.
```

Comment:
I'd also clarify here that you're making sure that the link is dynamically re-written